catalog-rest-service/src/main/antlr4/org/openmetadata/catalog/EntityLink.g4

```diff
@@ -0,0 +1,11 @@
+grammar EntityLink;
+
+entitylink: STARTPOINT SEPARATOR ENTITY_TYPE SEPARATOR ENTITY_FQN (SEPARATOR ENTITY_FIELD (SEPARATOR ENTITY_ATTRIBUTE)*)* ENDPOINT;
+
+ENTITY_TYPE: ('table'|'database'|'databaseSchema'|'metrics'|'dashboard'|'pipeline'|'chart'|'report'|'topic'|'mlmodel'|'bot'|'THREAD'|'location'|'glossary'|'glossaryTerm'|'tag'|'tagCategory'|'type'|'testDefinition'|'testSuite'|'testCase');
```

Comment:
@harshach i have added other entity types, do we have any specific text case that we want to add, 
for e.g. only lower case / only upper case?
as there are some upper case, camelCase 
the above was taken from Entity.java file